src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -3,17 +3,57 @@
 import edu.wpi.first.wpilibj.Preferences;
 
 public class MoPrefs {
-  static final double INTAKE_ROLLER_SETPOINT = 0.125;
-  static final double INTAKE_ROLLER_ACC_RAMP = 0.1;
-  static final int CLIMBER_ENCODER_LIMIT = 10;
-  static final double SHOOTER_HOOD_SETPOINT = 100;
-  static final double SHOOTER_HOOD_POSITION_TOLERANCE = 2;
-  static final double SHOOTER_GATE_SETPOINT = 1;
-  static final double SHOOTER_PID_SETPOINT = 4500;
-  static final double STORAGE_SPEED = 0.75;
-  static final double SHOOTER_FLYWHEEL_TOLERANCE = 100; // RPM
-  static final double SHOOT_FROM_WALL_HOOD_SETPOINT = 60; // The encoder setpoint that will reliably hit the Outer Port
-                                                          // when the robot is sitting up against the Power Port.
+
+  public static enum MoPrefsKey {
+    INTAKE_ROLLER_SETPOINT(0.125),
+
+    INTAKE_ROLLER_ACC_RAMP(0.1),
+
+    CLIMBER_ENCODER_LIMIT(10),
+
+    SHOOTER_HOOD_SETPOINT(100),
+
+    // FIXME: Copy this value to the new key
+    // The original key for this value was "SHOOTER_HOOD_POS_TOLERANCE"
+    // The new key is "SHOOTER_HOOD_POSITION_TOLERANCE"
+    SHOOTER_HOOD_POSITION_TOLERANCE(2),
+
+    SHOOTER_GATE_SETPOINT(1),
+
+    // FIXME: Copy this value to the new key
+    // The original key for this value was "Shooter PID Setpoint"
+    // The new key is "SHOOTER_PID_SETPOINT"
+    SHOOTER_PID_SETPOINT(4500),
+
+    STORAGE_SPEED(0.75),
+
+    // FIXME: Copy this value to the new key
+    // The original key for this value was "FLYWHEEL_TOLER"
+    // The new key is "SHOOTER_FLYWHEEL_TOLERANCE"
+    SHOOTER_FLYWHEEL_TOLERANCE(100), // RPM
+
+    // The encoder setpoint that will reliably hit the Outer Port
+    // when the robot is sitting up against the Power Port.
+    //
+    // FIXME: Copy this value to the new key
+    // The original key for this value was "Shoot From Wall Hood Setpoint"
+    // The new key is "SHOOT_FROM_WALL_HOOD_SETPOINT"
+    SHOOT_FROM_WALL_HOOD_SETPOINT(60);
+
+    private double defaultValue;
+
+    MoPrefsKey(double defaultValue) {
+      this.defaultValue = defaultValue;
+    }
+
+    public String getPrefsKey() {
+      return this.name();
+    }
+
+    public double getDefaultValue() {
+      return defaultValue;
+    }
+  }
 
   // PID constants for the shooter flywheel
   static final double SHOOTER_KP = 0.00005;
```

Comment:
Why are these values still hanging out here?  Did you want to validate the approach before you finished all the work?

Reply:
Pretty much. No point in putting in a lot of work converting the values if we decide we don't like this approach.